jekyll/_config.yml

```diff
@@ -14,7 +14,7 @@ plugins:
   - jekyll-sitemap
 
 algolia:
-  application_id: 06F2EYGKB6
+  application_id: U0RXNGRK45
```

Comment:
Is there an account name of some sort we can mention in a comment here to keep track?

Reply:
not exactly; multiple accounts can access the same information. I don't think a single account makes sense in that case, does it?